examples/kube-stack-private/variables.tf

```diff
@@ -18,7 +18,7 @@ variable "instance_type" {
 }
 
 variable "coreos_stable_ami_id" {
-  description = "set a stable coreos ami id"
+  description = "set a stable coreos ami id from https://goo.gl/2vJs7F"
```

Comment:
I would recommend http://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json as a more future-proof URL over http://stable.release.core-os.net/amd64-usr/1465.7.0/coreos_production_ami_hvm.txt

Reply:
Sure I'm updating the link